src/functions.js

```diff
@@ -166,6 +166,14 @@ defineFunction("\\color", {
     };
 });
 
+// \color is handled in Parser.js's parseImplicitGroup
```

Comment:
Thanks for commenting this.  It would be nice to have a way to mark functions in this file as to whether or not the parser needs to the parse an implicit group for them.